### PR TITLE
HTML Forms

### PR DESCRIPTION
**Pull request by louiewhitz**
https://user-images.githubusercontent.com/99847558/186218964-b33f1c9e-6db8-44eb-87b8-a0ae0f071c4c.mov

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  <select name="Select-an-option" id="select-question">
+    <option value="Select-an-option" selected>Select an option</option>
+    <option value="Interesting">Because it's interesting</option>
+    <option value="Fun">Because it's fun</option>
+    <option value="Powerful">Because it's powerful</option>
+  </select><br>
+  <label for="language">Favorite Language (choose one)</label><br>
+  <input type="radio" name="language" id="HTML" value="HTML">HTML
+  <input type="radio" name="language" id="CSS">CSS
+  <input type="radio" name="language" id="JavaScript">JavaScript <br>
```
Comment (BrettAlbright) [inline html-forms/index.html]: you want to use a label for each of the inputs here to make sure you have the ability to click on the label and have it activate the input.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
make sure you are using `div` elements to break the input onto multiple lines, and only the br tags if you need additional space. Looking good so far!

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Looking great! now lets try using some `div` elements to move the elements onto separate lines

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Looking much better! now the last thing is to make sure you have a `form` element wrapping all of the inputs!

**Comment by louiewhitz** [comment]
https://user-images.githubusercontent.com/99847558/186225819-318c8f7b-a389-4e3c-b58e-0961e9bb2585.mov

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**